Doc/library/sqlite3.rst

```diff
@@ -2465,6 +2467,28 @@ the context manager does nothing.
    # so the connection object should be closed manually
    con.close()
 
+``context.closing`` can also be used to perform singular transactions.
+In this case, the ``sqlite3`` connection is closed without an additional
+``sqlite.connect().close()`` after a context manager closes.
+
+For default ``sqlite`` context management cases which don't use
+``contextlib.closing``, this is intended to perform multiple transactions
+without fully closing the connection.
+
+A ``contextlib.closing`` example
+
+.. testcode::
+
+   import sqlite3
+   from contextlib import closing
+
+   with closing(sqlite3.connect("workfile.sqlite")) as cx:
+      with cx:
+        cx.execute("CREATE TABLE lang(id INTEGER PRIMARY KEY, name VARCHAR UNIQUE)")
+        cx.execute("INSERT INTO lang(name) VALUES(?)", ("Python",))
+
+
+   # no need to close the connection as contextlib.closing closed it for us
```

Comment:
IMO, all we need to do is to expand the note around line 2444:

- we should _link_ to `contextlib.closing` and succinctly mention that it can be used for "connection management" (or something like that)
- I'm not sure we need to repeat the `contextlib.closing` examples in the sqlite3 docs; the `contextlib` docs already provides a good example

Reply:
Thanks. I have worked on suggested changes.